docs/getting-started/motivation.md

```diff
@@ -29,7 +29,7 @@ The Feature Hub was designed with the following specific requirements in mind:
     the client.
   - The server-side rendered UI and its corresponding state must be hydrated on
     the client without visual impact.
-- Micro frontends that are incompatible with the integration environment should
+- Micro frontends that are incompatible with the Feature Hub environment should
```

Comment:
didn't we intentionally not mention the "Feature Hub" in this points? I recollect something along the lines of "keep the requirements abstracted from the solution".